netbox/templates/inc/table_htmx.html

```diff
@@ -15,12 +16,17 @@
                   <a href="#"
                      hx-get="{{ table.htmx_url }}{% querystring table.prefixed_order_by_field='' %}"
                      class="text-danger"
+                     {% if not table.embedded %}hx-push-url="true"{% endif %}
                      title="{% trans "Clear ordering" %}"
                   ><i class="mdi mdi-close"></i></a>
                 </div>
               {% endif %}
+              {% if table.filterset_form %}
+                {% include 'inc/table_header_filter_dropdown.html' with form_field=table.filterset_form|get_filter_field:column.name %}
+              {% endif %}
```

Comment:
Can we associate filter field names directly with their corresponding table columns? That would simplify things a bit.